tests/span_tests_qc.rs

```diff
@@ -0,0 +1,233 @@
+use quickcheck::Arbitrary;
+use tabled::{builder::Builder, object::Cell, Modify, Span, Style};
+
+#[derive(Clone, Debug)]
+struct TableStructure {
+    pub rows: Vec<Line>,
+    pub theme: ThemeFixture,
+    pub row_span: Vec<u8>,
+    pub col_span: Vec<u8>,
+}
```

Comment:
(Just a note);

See that you could use the same structure in both files.
I guess we could move it to a submodule and reuse it.

I was actually thinking about it for a while, to create a different   cargo crate for qc_tests.
Because they require additinal dependencies which increases build time.